mandatory/1-fix-functions.js

```diff
@@ -10,41 +10,37 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
+  if (isHappy===true) {
```

Comment:
since `isHappy` is a boolean, we don't need to compare it to `true` again. The original code `if (isHappy)` will also work.